### PR TITLE
Reason 3.5.0

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: packages/reason/reason.3.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+dev-repo: "git://github.com/facebook/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"         {>= "4.02" & < "4.09"}
+  "dune"          {>= "1.4"}
+  "ocamlfind"     {build}
```
Comment (kit-ty-kate) [inline packages/reason/reason.3.5.0/opam]: Are you sure ocamlfind is used anywhere? The only mentions I could find were related to esy and in `bspacks/reason_bspack.sh` but I'm not sure those two things are used here.

Comment (anmonteiro) [reply packages/reason/reason.3.5.0/opam]: Not sure, let me double check and get back to you.

File: packages/rtop/rtop.3.5.0/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+bug-reports: "https://github.com/facebook/reason/issues"
+dev-repo: "git://github.com/facebook/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"  {>= "4.02" & < "4.08"}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/rtop/rtop.3.5.0/opam]:
```suggestion
  "dune" {>= "1.1"}
```
The `dune-package` file says you require at least dune 1.1

**Comment by anmonteiro** [comment]
@kit-ty-kate Alright, I removed `ocamlfind` and made the suggested changes.

**Comment by anmonteiro** [comment]
I added `dune >= "1.4"` constraints because it's what we have in the `dune-project` file:
https://github.com/facebook/reason/blob/ea207004e021efef5a92ecd011d9d5b9b16bbded/dune-project#L1

**Comment by kit-ty-kate** [comment]
Oops I must have looked at the wrong project, my bad ^^"

**Comment by mseri** [comment]
Looks good, thanks

**Pull request merged**